src/nft.ts

```diff
@@ -169,14 +169,14 @@ export default class Nft extends FactoryBase {
   }
 
   /**
-   * Searches for AINFT objects created on the AIN Blockchain.
+   * Searches for AINFT objects created on the AIN Blockchain. This method accesses public data only.
```

Comment:
how about mentioning about the signature verification? e.g.
"This method accesses public data only and does not require signature in the requests."

Reply:
done!